weblab/core/processing.py

```diff
@@ -0,0 +1,14 @@
+from urllib.parse import urljoin
+
+from django.conf import settings
+
+def prepend_callback_base(url_path):
```

Comment:
I recall you were going to write a test for this; is that still todo or just forgot to add the file?

Reply:
the test is here: `core/tests/test_processing.py`